modules/openapi-generator/src/main/resources/cpp-qt5-client/api-body.mustache

```diff
@@ -123,6 +104,26 @@ void {{classname}}::setNetworkAccessManager(QNetworkAccessManager* manager) {
     _manager = manager;  
 }
 
+    /**
+     * Appends a new ServerConfiguration to the config mal for a specific operation.
+     * @param operation The id to the target operation.
+     * @param URL 
+     * @param variables A map between a variable name and its value. The value is used for substitution in the server's URL template.
+     * returns the index of the new server config on success and -1 if the operation is not found
+     */
+int {{classname}}::addServerConfiguration(const QString& operation, const QString& URL, const QString& description, const QMap<QString, {{prefix}}ServerVariable>& variables){
```

Comment:
even though what you have is the style I prefer, this whole file seems to typically put the reference operator touching the variable not the data type

Reply:
what data type? I`m not quiet sure what you need here. Every Operation has its own list of server configs. The are some servers stored double but its easier that way since I just loop through every operation once and create their own serverconfig list according to the openAPI file. You could change all this, so every server config is just stored once but those mustache templates can be tricky so i chose this way. It does not take up much memory since its just some strings.

Reply:
it is a nitpick/minor

the file mostly has the following style
```
const QString &operation
```
not
```
const QString& operation
```

Reply:
oh right. Yeah I might change that. But it does not make any difference as far as I know.

Reply:
You are right that it does not make any functional difference, but in enterprise or production level software, style and consistency count for something and help improve maintainability and lower the learning curve for new entrants to the codebase.